### PR TITLE
Increase the amount of supported IDs in the Info Command

### PR DESCRIPTION
**Pull request by greenbigfrog**
Just wrote a rough role info command, emoji are still missing. Not sure what else might be missing from the info command, those though I've personally wanted though or seen requested in the server.

### CODE REVIEW

File: backend/src/plugins/Utility/functions/getRoleInfoEmbed.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  const roleAge = humanizeDuration(Date.now() - role.createdAt, {
+    largest: 2,
+    round: true
+  });
+
+  let rolePermsArray : Array<string> = [];
+  for (const [x, _x] of Object.entries(role.permissions.json)) {
+    rolePermsArray.push(x.toString())
+  }
+  const rolePerms = rolePermsArray.join(', ')
```
Comment (0kku) [inline backend/src/plugins/Utility/functions/getRoleInfoEmbed.ts:46]: Consider doing this instead: 
```ts
const rolePerms = Object.keys(role.permissions.json).join(",");
```

File: backend/src/plugins/Utility/functions/getRoleInfoEmbed.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+  embed.fields.push({
+    name: preEmbedPadding + "Role information",
+    value: trimLines(`
+      Name: **${role.name}**
+      ID: \`${role.id}\`
+      Created: **${roleAge} ago** (\`${prettyCreatedAt}\`)
+      Position: ${role.position}
+      Color: #${role.color.toString(16).toUpperCase()}
```
Comment (0kku) [inline backend/src/plugins/Utility/functions/getRoleInfoEmbed.ts:55]: This slot likely should be 
```ts
role.color.toString(16).toUpperCase().padStart(6, "0")
```

File: backend/src/plugins/Utility/functions/getRoleInfoEmbed.ts [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    name: preEmbedPadding + "Role information",
+    value: trimLines(`
+      Name: **${role.name}**
+      ID: \`${role.id}\`
+      Created: **${roleAge} ago** (\`${prettyCreatedAt}\`)
+      Position: ${role.position}
+      Color: #${role.color.toString(16).toUpperCase().padStart(6, "0")}
+      Mentionable: ${role.mentionable}
+      Hoisted: ${role.hoist}
+      Managed: ${role.managed}
```
[the remark this answers is not in the document]
Comment (ThatsNoMoon) [reply backend/src/plugins/Utility/functions/getRoleInfoEmbed.ts:54]: Isn't information the point? Seems consistent with using `info` on a user ID.

[the remark this answers is not in the document]
Comment (almeidx) [reply backend/src/plugins/Utility/functions/getRoleInfoEmbed.ts:54]: What I was saying is that perhaps the "Managed" info is not necessary

**Comment by greenbigfrog** [comment]
current state

**Comment by almeidx** [comment]
Superseded by #198

**Pull request closed without merging**